### ISSUE TITLE
[DOCUMENTATION]: CDR2 - Adrian Holovaty (Frontend Javascript, backend logic, interactive interfaces)

### ISSUE THREAD

**Issue by aliceafanasieva** [fdnd-agency/embassyofthefreemind#242]
## Feedback we got:

- Use less frameworks, it will make it more readable. 
- Minimise javascript (loading page slower)
- Page load and appear should be faster 
- Make more side projects
